freeswitch/scripts/astpp/scripts/astpp.xml.lua

```diff
@@ -200,6 +200,19 @@ function freeswitch_xml_outbound(xml,destination_number,outbound_info,callerid_a
     return xml
 end
 
+--To split the DID destination number string
+function split_numbers(source, delimiters)
+        local elements = {}
+        local pattern = '([^'..delimiters..']+)'
+        string.gsub(source, pattern, function(value) elements[#elements + 1] =     value;  end);
+        return  elements
+end
+function split_delis(source, delimiters)
```

Comment:
Move split_number and split_detais functions to astpp_functions module as those doing logic work on string. astpp.xml.lua must have only xml dialplan related code.

Reply:
@smrdoshi Functions for splitting destination strings are removed from astpp.xml.lua